AnaClos.HabitTracker/AnaClos.HabitTracker/Program.cs

```diff
@@ -0,0 +1,337 @@
+﻿using AnaClos.HabitTracker;
+using Microsoft.Data.Sqlite;
+using System.Globalization;
+
+
+string choice = "q";
+string connectionString = @"Data Source=habit_Tracker.db";
+
+CreateBaseAndTable(connectionString);
+
+choice =Menu();
```

Comment:
Inconsistent spacing. `choice = Menu();` Also if Menu is returning something the name of the method should be clearer.